Assets/HoloToolkit/Input/Scripts/Gaze/GazeManager.cs

```diff
@@ -300,14 +299,16 @@ private RaycastResult FindClosestRaycastHitInLayerMasks(List<RaycastResult> cand
                 combinedLayerMask = combinedLayerMask | layerMaskList[i].value;
             }
 
-            var canvasCandidates = candidates.Where(c => c.gameObject != null)
-                .Where(c => IsLayerInLayerMask(c.gameObject.layer, combinedLayerMask))
-                .Select(c => new CanvasRaycastResult(c))
-                .DefaultIfEmpty(CanvasRaycastResult.Empty)
-                .ToList();
-
+            var canvasCandidates = new List<CanvasRaycastResult>();
```

Comment:
Does it make sense if we made this a private field and just clear the list each time, instead of allocating a new list?